### PR TITLE
Add warning and MCU family declaration

### PR DESCRIPTION
**Pull request by Emanuel-Allely**

### CODE REVIEW

**Review by Simonbdy — APPROVED** [review]

File: src/pages/embedded/dev_env/platformio.md [hunk 1]
```diff
@@ -10,8 +12,11 @@
 ```Json
 lib_deps = Luos
 board = <board name>
+-DLUOSHAL= <MCU family>
```
Comment (nicolas-rabault) [inline src/pages/embedded/dev_env/platformio.md:15]:
```
build_flags =
    -DLUOSHAL= <LuosHAL folder>
```

File: src/pages/embedded/dev_env/platformio.md [hunk 2]
```diff
@@ -10,8 +12,11 @@
...
 ```
-Replace `<board name>` with the name of the board you are using, eg. `board = l0` for the L0 board.
+Replace `<board name>` with the name of the board you are using, eg. `board = l0` for the L0 board or `board = disco_f030r8` for the STM32 F0308 Discovery board
+
+Replace `<MCU family>` with the family of the MCU you are usign, eg. `-DLUOSHAL=STM32F0` for the F0 family.
```
Comment (nicolas-rabault) [inline src/pages/embedded/dev_env/platformio.md:19]: Replace `<LuosHAL folder>` with the <a href="https://github.com/Luos-io/LuosHAL">Luos HAL</a> folder corresponding to the family of the MCU you are usign, eg. `-DLUOSHAL=STM32F0` for the F0 family.

**Review by nicolas-rabault — CHANGES_REQUESTED** [review]

**Review by Emanuel-Allely — COMMENTED** [review]
I did the change

**Review by nicolas-rabault — APPROVED** [review]
ok, now you have to fixup your commit and merge ;)

**Pull request merged**